src/components/Trophy.js

```diff
@@ -16,6 +16,7 @@ class Trophy extends React.Component {
   }
 
   createTrophy(){
+    // There is a lot of repetition here - see if you can refactor it out - Harold
```

Comment:
We will take a look at a possible refactor here.  The trick is that we have these unique award zones and each collection has a unique image for those "zones".  We will try out best to refactor. (might need advice on this one)